### PR TITLE
Added aliases for in dnszone module arguments

### PR DESCRIPTION
**Pull request by seocam**

### CODE REVIEW

File: README-dnszone.md [hunk 1]
```diff
@@ -161,6 +161,7 @@ ipadnszone
 
 Variable | Description | Required
 -------- | ----------- | --------
+`name` \| `zone_name` | The zone name string. | yes
```
Comment (rjeffman) [inline README-dnszone.md]: Only to keep it more consistent with other modules, move it to line 166, after `ipaadmin_*` itens. Also, add this change to the plugins/ipadnszone.py file (under DOCUMENTATION).

File: README-dnszone.md [hunk 2]
```diff
@@ -172,7 +173,7 @@ Variable | Description | Required
 `name_server`| Authoritative nameserver domain name | no
 `admin_email`| Administrator e-mail address | no
 `update_policy`| BIND update policy | no
-`dynamic_update`| Allow dynamic updates | no
+`dynamic_update` \| `dynamicupdate` | Allow dynamic updates | no
```
Comment (rjeffman) [inline README-dnszone.md]: Add this change to the plugins/ipadnszone.py file (under DOCUMENTATION).

File: README-dnszone.md (hunk 1 above)
Comment (seocam) [reply README-dnszone.md]: Fixed.

File: README-dnszone.md (hunk 2 above)
Comment (seocam) [reply README-dnszone.md]: Done.

**Pull request merged**